app/services/requisicion_service.py

```
@@ -0,0 +1,536 @@
+            )
+            current_app.logger.info(
+                f"Correo enviado a {requisicion.correo_solicitante} con estado {nuevo_estado}"
+            )
+
+            if nuevo_estado == ESTADO_INICIAL_REQUISICION: # Aunque raro cambiar A este estado, se mantiene la lógica
+                mensaje_almacen = generar_mensaje_correo(
+                    "Almacén", requisicion, nuevo_estado, comentario or ""
+                )
+                enviar_correos_por_rol(
+                    "Almacen", "Nueva requisición pendiente", mensaje_almacen
+                )
```

Pylint (3.3.7)

[error] No value for argument 'UsuarioModel' in function call

(E1120)